chooloolib/src/main/java/com/chooloo/www/chooloolib/adapter/PhonesAdapter.kt

```diff
@@ -26,5 +29,5 @@ class PhonesAdapter @Inject constructor(
         }
     }
 
-    override fun convertDataToListData(data: List<PhoneAccount>) = ListData.fromPhones(data)
+    override fun convertDataToListData(data: List<PhoneAccount>) = ListData.fromPhones(data, context)
```

Comment:
First of all, I'm sorry for my mistakes (especially about creating a static instance of the app as a way to get `context`). Would the changes I have made be alright?

Reply:
Yes looks great, just please remove the static app cotext and the Strings object